src/components/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FC } from 'react';
+
+type Props = {
+  hasError: string,
+  setHasError: (string: string) => void,
```

Comment:
Consider renaming the parameter 'string' to a more descriptive name like 'error' in the 'setHasError' function type definition. 😊